src/Http/Routing/src/Logging/RouterMiddlewareLoggerExtensions.cs

```diff
@@ -14,7 +14,7 @@ static RouterMiddlewareLoggerExtensions()
         {
             _requestDidNotMatchRoutes = LoggerMessage.Define(
                 LogLevel.Debug,
-                1,
+                new EventId(1, "RequestDidNotMatchRoutes"),
```

Comment:
I guess one of these has to change to match verb tenses

Reply:
Updated